python/lsst/cell_coadds/_single_cell_coadd.py

```diff
@@ -85,8 +85,7 @@ def __init__(
         self._common = common
         # Remove any duplicate elements in the input, sorted them and make
         # them an immutable sequence.
-        # TODO: Remove the conditioning in DM-40563.
-        self._inputs = tuple(sorted(set(inputs))) if inputs else ()
```

Comment:
I'm intentionally dropping support for files generated with previous versions (since we are still in version 0.x) since I feel that it's easier to drop support for no inputs rather than maintaining this until we hit `1.0`.